web/src/pages/Route/components/Step1/MatchingRulesView.tsx

```diff
@@ -136,9 +136,18 @@ const MatchingRulesView: React.FC<RouteModule.Step1PassProps> = ({
           case '~~':
             renderText = formatMessage({ id: 'page.route.regexMatch' });
             break;
+          case '~*':
+            renderText = formatMessage({ id: 'page.royte.caseInsensitiveRegexMatch' });
+            break;
           case 'IN':
             renderText = formatMessage({ id: 'page.route.in' });
             break;
+          case 'HAS':
```

Comment:
How about try to use a cont object or enum to define the the relationship between the operator and its show name? this can also be reused in https://github.com/apache/apisix-dashboard/pull/2022/files#diff-a55e5a5bd491758bf28203f2e3eb5ada3d01944348e00c2cafb6671e79621bc3R288-R298

Reply:
Think it's good, let me try